### ISSUE TITLE
Is it necessary to transform the values of VOC masks to range [0,20]？

### ISSUE THREAD

**Issue by KleinXin** [Tramac/awesome-semantic-segmentation-pytorch#28]
THX

**Comment by Tramac** [comment]
Yes，the number of classes is 21 for VOC.

**Comment by KleinXin** [comment]
How do you initialize the weights of decoders?
I used kaiming_normal_

**Comment by Tramac** [comment]
It is a hyperparameter, you can use ```normal```,```kaiming_normal_```,```xavier``` or others. I'm not sure which is better.

**Comment by KleinXin** [comment]
thx
